### DISCUSSION TITLE
Bank Tag Layout Addition - (Shift Click) Tag-Inventory

### DISCUSSION THREAD

**Discussion by Kuemper35** [runelite/runelite#17401] [Ideas]
Context:
Utilizing the "Bank Tag Layouts" plug-in to store clue scrolls inventory tags.

Issue:
Banking a newly obtained clue scroll does not auto-populate a clue scroll inventory tag when choosing the option "Deposit Inventory." However, it does auto-populate if you "Tag-inventory" before "Deposit Inventory." Currently, our only method to "Tag-inventory" is by Right Clicking w/ your mouse & manually choosing the option.

https://github.com/runelite/runelite/assets/157139224/2b294345-8e33-4449-8ed8-43c05a37db47

Resolution:
Add an addition to the Bank Tag Layout plug-in OR Menu Entry Swapper plug-in that allows players to choose their (Shift Click) option to "Tag-inventory" when clicking the "Deposit Inventory" function in the bank UI.

![Bank Tag LAyout - Clue Scroll](https://github.com/runelite/runelite/assets/157139224/34518c6a-2c0f-49b3-94d9-e21283d0207c)

**Comment by raiyni** [comment]
Support for hub plugins should be directed to the author of the plugin.
You can find the support link by searching for the plugin in the Plugin Hub panel and clicking the ? button on the plugin, or by right-clicking the plugin in the plugin panel and clicking the Support menu option.

**Comment by Alexsuperfly** [comment]
each clue is a different id, in order for every one of them to be tagged you need to "variation tag" them by either appending a `*` to the tag name on the item or shift dragging it into the tag tab, this will allow all variations of that item to also have that tag, like barrows armor that degrades and clue scrolls
